.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+**Summary:** Summary of changes
+
+Addresses [VEDA-XX: Develop amazing new feature](https://github.com/NASA-IMPACT/veda-data-pipelines/issues/XX)
+
+## Changes
+
+* Detailed list or prose of changes
+* ...
+
+## PR Checklist
+
+- [ ] Update CHANGELOG
+- [ ] Unit tests
+- [ ] Ad-hoc testing - Deploy changes and test manually
+- [ ] Integration tests
```

Comment:
kinda ironic that you've added this template, but the PR doesn't follow it 😂

Reply:
It will be effected after the merge, right now there is no PR template